src/server/test/web/readingsCompareMeterQuantity.js

```diff
@@ -158,8 +158,114 @@ mocha.describe('readings API', () => {
 
 				// Add C10 here
 
-				// Add C11 here
-
+				mocha.it('C11: 1 day shift end 2022-10-31 17:00:00 for 15 minute reading intervals and quantity units & kWh as BTU reverse conversion', async () => {
+					const unitData = [
+						// adding units u1, u2, u3, u16
+						{
+							//u1
+							name: 'kWh', 
+							identifier: '', 
+							unitRepresent: Unit.unitRepresentType.QUANTITY, 
+							secInRate: 3600, 
+							typeOfUnit: Unit.unitType.UNIT, 
+							suffix: '', 
+							displayable: Unit.displayableType.ALL, 
+							preferredDisplay: true, 
+							note: 'OED created standard unit'
+						},
+						{
+							//u2
+							name: 'Electric_Utility',
+							identifier: '',
+							unitRepresent: Unit.unitRepresentType.QUANTITY,
+							secInRate: 3600,
+							typeOfUnit: Unit.unitType.METER,
+							suffix: '',
+							displayable: Unit.displayableType.NONE,
+							preferredDisplay: false,
+							note: 'special unit'
+						},
+						{
+							// u3
+							name: 'MJ', 
+							identifier: 'megaJoules', 
+							unitRepresent: Unit.unitRepresentType.QUANTITY, 
+							secInRate: 3600, typeOfUnit: Unit.unitType.UNIT, 
+							suffix: '', displayable: Unit.displayableType.ALL, 
+							preferredDisplay: false, 
+							note: 'MJ'
+						},
+						{
+							// u16
+							name: 'BTU', identifier: '', 
+							unitRepresent: Unit.unitRepresentType.QUANTITY, 
+							secInRate: 3600, 
+							typeOfUnit: Unit.unitType.UNIT, 
+							suffix: '', displayable: Unit.displayableType.ALL, 
+							preferredDisplay: true, 
+							note: 'OED created standard unit'
+						},
+					];
+					const conversionData = [
+						// adding conversions c1, c2, c3
+						{
+							// c1
+							sourceName: 'Electric_Utility', 
+							destinationName: 'kWh', 
+							bidirectional: false, 
+							slope: 1, 
+							intercept: 0, 
+							note: 'Electric_Utility → kWh' 
+						},
+						{
```

Comment:
By looking at this code I realized that the description was off for the conversions (it is now fixed). It should have specified c1, c6, c3 to get the reversed conversion. I'm sorry about this but could you update (B11 has it the desired way).